src/cli/BaseCLIController.ts

```diff
@@ -88,4 +89,18 @@ export class BaseCLIController {
     const { error } = await this.execDvc('--version')
     return !error
   }
+  
+  public requiredPackageVersion = '5.2.1'
+  public packageName = '@devcycle/cli'
+
+  public async isCliMinVersion() {
+    const { error, output } = await this.execShell(`npm list -g ${this.packageName} --depth=0 --json`)
```

Comment:
could we get the version using `dvc --version`? since a user may not have npm if they installed the cli using homebrew

Reply:
Im pretty sure installing through homebrew will use npm under the hood, so this should work still. It still might be better to just use our own version command though so ill make this change